CMakeLists.txt

```diff
@@ -178,50 +178,49 @@ endif()
 include_directories("${PROJECT_BINARY_DIR}/config")
 include_directories("${PROJECT_SOURCE_DIR}/src")
 
+# Systemwide (non-bundled) dependencies are usually not provided
+# as static libraries, so we can just switch this on by default.
+option(USE_STATIC "Use static dependencies" ON)
```

Comment:
I was trying hard to avoid doing this for two reasons.  First, now it's another knob distros need to be aware of.  Second, because `OFF` doesn't make sense for when we use bundled dependencies (we don't install the dependent libraries anywhere useful).  By only having static libraries available from the third party build, we'd get the behavior we wanted, and things fell back to shared libraries without explicitly asking for it.  That was the hope anyways.